internal/nostr/nostr.go

```diff
@@ -669,6 +666,26 @@ func (svc *Service) processEvents(ctx context.Context, subscription *Subscriptio
 			"subscriptionId": subscription.ID,
 		}).Info("Received EOS")
 		receivedEOS = true
+
```

Comment:
Regarding @bumi 's comment

This method feels more straightforward than the other one that handles both.

Reply:
tag handling looks wrong:
```
tags := new(nostr.TagMap)
*tags = make(nostr.TagMap)
```

Reply:
Fixed